src/components/FaqSection/Faq/Faq.styles.js

```diff
@@ -0,0 +1,116 @@
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+
+const Heading = styled.div`
+  ${tw`
+        grid
+        w-100
+        grid-rows-1
+        gap-4
+    `}
+`
+
+const FaqContainer = styled.div`
+  ${tw`
+        bg-secondary
+        relative
+        p-2
+        rounded-lg
+   `}
+`
+
+const Section = styled.div`
+  ${tw`
+        grid
+        grid-flow-col 
+        grid-cols-2
+        w-full
+        gap-4
+        mt-4
+        text-white
+    `}
+`
+
+const Bot = styled.div`
+  ${tw`
+        m-4
+        flex
+    `}
+`
+
+const Chat = styled.div`
+  ${tw`
+        ml-3
+        flex
+        items-center
+        justify-center
+        bg-purple-light
+        rounded-full
+        relative
+        text-black
+        border-2 border-gray-800
+    `}
+`
+
+const Bodydark = styled.p`
```

Comment:
I am not sure if we have created this typography, but we surely have a body typography in shared folder. Check once.